src/arche/tools/api.py

```diff
@@ -139,10 +139,12 @@ def get_items_with_pool(
     processes_count = min(max(helpers.cpus_count(), workers), active_connections_limit)
     batch_size = math.ceil(count / processes_count)
 
+    start_idxs = [i for i in range(start_index, start_index + count, batch_size)]
```

Comment:
`list(range(start_index, start_index + count, batch_size))`.
But if does not need to be a list, then just `range